### ISSUE TITLE
Missing alternative texts for images

### ISSUE THREAD

**Issue by Comandeer** [ckeditor/ckeditor4#5064]
## Type of report

Docs

## Provide a description of requested docs changes

Some examples, e.g. [Applying Styles to Editor Content](https://ckeditor.com/docs/ckeditor4/latest/examples/styles.html) or [Basic Text Styles: Bold, Italic and More](https://ckeditor.com/docs/ckeditor4/latest/examples/basicstyles.html), contain images representing elements of the editor's UI (in this case Styles combo and basic styles buttons). However, these images lack alternative text. They are not totally inaccessible (due to having the `[title]` attribute) but the alternative text in the `[alt]` attribute should be added.

Probably the best way to fix it would be to slightly rephrase the docs (texts in brackets indicate the place of the image alongside the proposed alternative text):

before:

> When the Styles Combo plugin is enabled, the [Styles] button is automatically added to the toolbar.

after:

> When the Styles Combo plugin is enabled, the Styles button [] is automatically added to the toolbar.

 In such a case the name of the button is provided directly in the text so the image [should have an empty alternative text](https://html.spec.whatwg.org/multipage/images.html#a-graphical-representation-of-some-of-the-surrounding-text) as it's added only as an additional cue, duplicating the already stated information.
